Bottom: bdd5cbb8e3600ea0567b9fcc228837a97e2e5263
Top:    82ddbac09bb4c093ee924d2b2674af813bf3fb36
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-05 12:30:29 -0500

Use version sort instead of glob

---

diff --git a/farm-compile b/farm-compile
index 0ded131..b8220c4 100755
--- a/farm-compile
+++ b/farm-compile
@@ -14,7 +14,7 @@ if test -d utils; then
 fi
 
 error=
-for bintree in $farmbin/*; do
+for bintree in $(ls -d $farmbin/* | sort -V); do
 	arch=
 	if test -r $bintree/configured-i386; then
 		arch=i386
diff --git a/farm-config32 b/farm-config32
index 06b08e2..cdfab13 100755
--- a/farm-config32
+++ b/farm-config32
@@ -6,7 +6,7 @@ set -e
 arch=i386
 ccflags=-m32
 
-for srctree in $farmsrc/*; do
+for srctree in $(ls -d $farmsrc/* | sort -V); do
 	test -d $srctree/Documentation || continue
 	bintree=$farmbin/$(basename $srctree)-$arch
 	echo -n "Configuring $bintree "
diff --git a/farm-config64 b/farm-config64
index 91195ea..38f80ab 100755
--- a/farm-config64
+++ b/farm-config64
@@ -6,7 +6,7 @@ set -e
 arch=x86_64
 ccflags=-m64
 
-for srctree in $farmsrc/*; do
+for srctree in $(ls -d $farmsrc/* | sort -V); do
 	test -d $srctree/Documentation || continue
 	bintree=$farmbin/$(basename $srctree)-$arch
 	echo -n "Configuring $bintree "
